content/docs/command-reference/import.md

```diff
@@ -78,7 +78,8 @@ from the source repo.
 - `-o <path>`, `--out <path>` - specify a path (directory and/or file name) to
   the desired location to place the imported file in (instead of using the
   current working directory). If an existing directory is specified, the target
-  data will be placed inside.
+  data will be placed inside. If `path` contains a parent directory which does
+  not exist, this command will fail.
```

Comment:
Sorry for late review @pmrowla, I have a Q:

> If `path` contains a parent directory

What does that mean? "Containing a parent" sounds contradictory. Parents contain you 😋

Reply:
Plus how can you contain something that does not exist? Then you don't contain it... Sorry, just confused by this.

Reply:
If you run `dvc import -o foo/bar` and `foo/` doesn't already exist, `import` will fail

Reply:
I see, got it. I may reword a little. Thanks Peter!